### PR TITLE
Primitives and clean up required for example contracts

### PR DESCRIPTION
**Pull request by KentShikama**

### CODE REVIEW

File: src/Ob.cc [hunk 1]
```diff
@@ -1460,6 +1463,18 @@ DEF("cwd",sysCwd, 0, 0)
 	return PRIM_ERROR(buf);
 }
 
+// TODO: Swap with something crypotgraphically secure and uniform in length
+DEF("random-number-init", randomNumberInit, 0, 0)
+{
+    srand(time(NULL));
+    return NIV;
+}
+DEF("random-number",randomNumber, 0,0)
```
Comment (ghost) [inline src/Ob.cc]: Put a carriage return between definitions and a space after commas.

File: src/RBLstream.cc [hunk 2]
```diff
@@ -364,6 +366,28 @@ DEF("prim-print",obPrint, 1, MaxArgs)
...
+        if (errno = fprintf(strm->stream, "%s", buf))
+            return FIXNUM(-errno);
+        else
+            return NIV;
+    } else {
+        return PRIM_ERROR("cannot print on closed ostream");
+    }
+}
+
+
```
Comment (ghost) [inline src/RBLstream.cc:390]: leave a single line, not two

**Review by ghost — APPROVED** [review]
LGTM modulo whitespace

File: src/RBLstream.cc (hunk 2 above)
Comment (KentShikama) [reply src/RBLstream.cc:390]: Two seems consistent with the rest of the file.

Comment (ghost) [reply src/RBLstream.cc:390]: You're right, sorry; I just saw a nearby one was one line and assumed.

**Pull request closed without merging**